design-documents/graph-ql/coverage/catalog/compare-list.graphqls

```diff
@@ -10,11 +10,11 @@ type ComparableItem {
 
 type ComparableAttribute {
     code: String! @doc(description: "Attribute code ")
-    title: String! @doc(description: "Addibute display title")
+    title: String! @doc(description: "Attribute display title")
 }
 
 type CompareList {
-    list_id: ID! @doc(description: " Compare list id")
+    uid: ID! @doc(description: "Compare list id")
     items: [ComparableItem] @doc(description: "Comparable products")
```

Comment:
Would be good to make the following changes to ComparableItem in this PR
1. Rename productId to productUid and introduce uid on the ProductInterface
2. Rename priceRange to price_range
3. Can we have media_gallery: [MediaGalleryInterface]! instead of images: [ProductImage]!?

Reply:
Agree on 2 and 3. I will update the document.  I think @paliarush exposed the id for the same reason on #4. I am okay with exposing ProductInterface, as I don't foresee N + 1 concerns here.

Reply:
Figured that the reason for #3 ProductImage is, MediaGalleryInterface encompasses even videos. Videos are not supported in Product compare.